Content/ResourceLoader/FormSelectionResourceLoader.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\FormBundle\Content\ResourceLoader;
+
+use Sulu\Bundle\FormBundle\Form\BuilderInterface;
+use Sulu\Content\Application\ResourceLoader\Loader\ResourceLoaderInterface;
+use Symfony\Component\Form\FormView;
+
+final class FormSelectionResourceLoader implements ResourceLoaderInterface
```

Comment:
Should be only `FormResourceLoader` the resourceloader is independent of the propertyresolver and therefore should be used for both single and multi selections.